[DOCS-3376] Connect RUM and Traces Edits (#13633)
DataDog/documentation: content/en/real_user_monitoring/connect_rum_and_traces/_index.md

    applicationId: '<DATADOG_APPLICATION_ID>',
    clientToken: '<DATADOG_CLIENT_TOKEN>',
    ...otherConfig,
    service: "my-web-application",
    allowedTracingOrigins: ["https://api.example.com", /https:\/\/.*\.my-api-domain\.com/]
})
```

> **Note**: `allowedTracingOrigins` accepts Javascript String and RegExp that should match the origins called by your browser application, with origin defined as: `<scheme> "://" <hostname> [ ":" <port> ]`
> **Note**: `service` is a required field to tie RUM into traces

Reviewer: ```suggestion
To connect RUM to Traces, you need to specify your browser application in the `service` field.

**Note**: `allowedTracingOrigins` accepts Javascript strings and RegExp that matches the origins called by your browser application, defined as: `<scheme> "://" <hostname> [ ":" <port> ]`.
```
